src/testnet/helium/node.rs

```diff
@@ -64,14 +66,36 @@ pub struct Node {
     burnchain_tip: Option<BlockSnapshot>,
     pub chain_state: StacksChainState,
     chain_tip: Option<StacksHeaderInfo>,
-    config: Config,
+    pub config: Config,
     keychain: Keychain,
     last_sortitioned_block: Option<SortitionedBlock>,
     mem_pool: MemPoolFS,
     event_dispatcher: EventDispatcher,
     nonce: u64,
 }
 
+pub const DEFAULT_CONNECTION_OPTIONS: ConnectionOptions = ConnectionOptions {
```

Comment:
Can we keep this in `ConnectionOptions::default()`?

Reply:
These would be defaults specific to the helium::node.

Reply:
Ah, thanks for clarifying!